docker/server/index.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express')
+
+const defaults = {
+  port: 8080,
+  host: 'http://localhost:3000/graphql',
+  title: 'GraphQL Playground'
+}
+
+const port = +process.env.GRAPHQLPLAYGROUND_PORT || defaults.port
+const host = process.env.GRAPHQLPLAYGROUND_HOST || defaults.host
+const title = process.env.GRAPHQLPLAYGROUND_TITLE || defaults.title
+
+const app = express()
+
+const html = `
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset=utf-8/>
+  <meta name="viewport" content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, minimal-ui">
+  <title>${title}</title>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
+  <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
+  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js"></script>
```

Comment:
shouldn't those come from the image ? (if I'm running graphql-playground in docker I expect it to work even without internet connection)